fast64_internal/f3d/f3d_gbi.py

```diff
@@ -133,6 +134,14 @@ def isUcodeF3DEX3(F3D_VER: str) -> bool:
     return F3D_VER == "F3DEX3"
 
 
+def is_ucode_t3d(UCODE_VER: str) -> bool:
+    return UCODE_VER == "T3D"
+
+
+def is_ucode_f3d(UCODE_VER: str) -> bool:
+    return UCODE_VER not in {"T3D"}
```

Comment:
you could do ``return not is_ucode_t3d(UCODE_VER)`` but I'd understand if you don't like it

Reply:
no because in the future we could have more non f3d ucodes, maybe a model64 option for barebones materials

Reply:
oh yeah good point